index.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,900,700' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
```

Comment:
Can you include these in the CSS file instead? This file is only used for mockup-testing.